src/Lecture1.hs

```diff
@@ -104,7 +121,16 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+mySum :: Int -> [Int] -> Int
+mySum result myList = 
+    if null myList
+    then result
+    else result + (head myList) + mySum result (tail myList)
```

Comment:
There's a standard function `sum` to find a sum of all elements in the list. You can use it instead of implementing your own function.

Additionally, in your implementation, I see that the `result` variable is a bit redundant. You don't change it during the recursion and it's always `0` from the start, so it doesn't affect the outcome of this function.